### DISCUSSION TITLE
Change pagination size globally [Sep 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6461] [Discord Threads]
> 👨‍🚀 **wingkwong**

**Comment by o1lab** [comment]
> 🙋‍♂️ **tinhochu**  
#### How can I do that?  
#  
> 👨‍🚀 **wingkwong**  
#### see https://github.com/nocodb/nocodb/discussions/942  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1141632026584305664)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
